src/utils/registrable.beneficial.owner.ts

```diff
@@ -25,7 +26,7 @@ export const getRegistrableBeneficialOwner = (req: Request, res: Response, next:
     return res.render(templateName, {
       backLinkUrl: backLinkUrl,
       templateName: templateName,
-      appData,
+      ...appData,
```

Comment:
the ...appData change here will also affect the update screen, I'm guessing the update version of this screen had been tested previously and working ok so this change might break it?

Reply:
We need this because the company name and number do not display when there are errors on the page.